Runtime/Shaders/MapsBaker.cs

```diff
@@ -98,9 +100,22 @@ internal static class MapsBaker {
             FindTexture(MeshSyncConstants._MetallicGlossMap, textureHolders, materialProperties,
                 MeshSyncConstants._Metallic, 0, out var metalTexture);
 
-        texturesExist |=
-            FindTexture(MeshSyncConstants._GlossMap, textureHolders, materialProperties,
-                MeshSyncConstants._Glossiness, 1, out var glossTexture);
+        Texture2DDisposable glossTexture;
+        bool                usingRoughness = false;
+        // If there is a roughness map, use that instead of smoothness and invert it:
+        if (materialProperties.ContainsKey(MeshSyncConstants._RoughMap)) {
```

Comment:
This is duplicate with the in line 187 below, which seems to be located in the same function ?
Would it be possible to eliminate duplicates ?